### PR TITLE
[48] [UI] As a user, I can see the product information section on the Product Detail screen

### LINKED ISSUE #48: [UI] As a user, I can see the product information section on the Product Detail screen
## Why

As a user, I can see the product information section on the Product Detail screen.
 
## Acceptance Criteria

The product information section on the Product Detail screen contains:

- Product image on the top
- Product name label
- Product price and currency label
- "Add to cart" button 
- A separated line between the "Add to cart" button and "About" title label
- "About" title label
- Product description label with multiple lines

## Resource

image

### PR DESCRIPTION
**Pull request by nmint8m**
https://github.com/nimblehq/nimble-ecommerce-ios/issues/48

## What happened 👀

As a user, I can see the product information section on the Product Detail screen.

The product information section on the Product Detail screen contains:

- Product image on the top
- Product name label
- Product price and currency label
- "Add to cart" button
- A separated line between the "Add to cart" button and "About" title label
- "About" title label
- Product description label with multiple lines 
 
## Insight 📝

In this PR, I implement the product information section on the Product Detail screen.
 
## Proof Of Work 📹

image

### CODE REVIEW

File: ecommerce-ios/Views/ProductInformation/ProductInformationView.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+            Button(
+                action: { print("did tap add to cart button") },
+                label: {
+                    Text("Add to cart")
+                        .font(.system(size: 17.0).weight(.semibold))
+                        .foregroundColor(.white)
+                }
+            )
+            .frame(width: screenWidth - 32.0, height: 50.0)
```
Comment (blyscuit) [inline ecommerce-ios/Views/ProductInformation/ProductInformationView.swift]: what do you think about
```

            .frame(maxWidth: .infinity, minHeight: 50.0)
            .padding(.horizontal, 32.0)
```
so we can make these view more responsive.

**Review by vnntsu — APPROVED** [review]
LGTM!

File: ecommerce-ios/Views/ProductInformation/ProductInformationView.swift (hunk 1 above)
Comment (nmint8m) [reply ecommerce-ios/Views/ProductInformation/ProductInformationView.swift]: It's great to know it.

**Review by blyscuit — APPROVED** [review]
👍

**Pull request merged**